### PR TITLE
NIP-11: add optional image

### PR DESCRIPTION
**Pull request by suhailsaqan**
Add optional image to Relay Information Document.

### CODE REVIEW

**Comment by jeffthibault** [comment]
LGTM

**Review by alexgleason — APPROVED** [review]

**Comment by alexgleason** [comment]
I've been thinking for a while we should have this. cc @vitorpamplona

**Comment by vitorpamplona** [comment]
Amethyst uses `/favicon.ico` on the relay address to get the image. 

But this could also work. It's all about compliance. 

We show the relay icon upfront because we want to give relays traffic. But most relays don't bother to add a logo... :(

**Review by fiatjaf — APPROVED** [review]

File: 11.md [hunk 1]
```diff
@@ -255,6 +255,18 @@ Relays that require payments may want to expose their fee schedules.
 }
 ```
 
+### Image ###
+
+A URL pointing to an image to be used to represent the relay.
```
Comment (alexgleason) [inline 11.md:260]: It occurred to me this isn't very specific, which could cause the field to be used in competing ways by different applications. Is the use-case an icon, like I imagine? If so I would call it "icon" and recommend it should be square. That would leave room for us to include fields like "banner" in the future.

Comment (vitorpamplona) [reply 11.md:260]: Agreed.

Comment (suhailsaqan) [reply 11.md:260]: True. Will change now.

**Comment by alexgleason** [comment]
I think this is good to go. 👍

**Review by vitorpamplona — APPROVED** [review]

**Pull request merged**